inc/ticket.class.php

```diff
@@ -105,26 +107,28 @@ static function getAllForTicket($ID, $start = 0, $limit = 0, $sqlfilter = '') {
     * @param $ID           integer     plugin_credit_entities_id ID
     * @param $start        integer     first line to retrieve (default 0)
     * @param $limit        integer     max number of line to retrive (0 for all) (default 0)
-    * @param $sqlfilter    string      to add a SQL filter (default '')
+    * @param $sqlfilter    array       to add a SQL filter (default '')
     * @return array of vouchers
    **/
-   static function getAllForCreditEntity($ID, $start = 0, $limit = 0, $sqlfilter = '') {
+   static function getAllForCreditEntity($ID, $start = 0, $limit = 0, $sqlfilter = []): array {
```

Comment:
`$start`, `$limit` and `$sqlfilter` are never set in the only existing call to this method. These parameters could be dropped and related code could be cleaned.